Apps/napi-jsi/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# cmake 3.15.3+ because of a c++ regression when swift is enable
+cmake_minimum_required(VERSION 3.15.3)
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+
+project(napi-jsi)
+
+# Shared component source will be built in this UWP flavor
+# We can support other flavors if the need arises, but a UWP build should get us the first wave of napi-jsi build coverage
+set(BABYLON_NATIVE_PLATFORM "UWP")
+set(CMAKE_SYSTEM_NAME "WindowsStore")

```

Comment:
Should we predefine the `CMAKE_SYSTEM_NAME`? Isn't this usually inferred or overridden from the command line?

Reply:
Updated the pipeline definition. it seems like UWP doesn't get resolved on its own, but if we specify it to cmake as an argument we dont need these definitions.